test/parallel_api/algorithm/alg.modifying.operations/replace.pass.cpp

```diff
@@ -122,7 +122,7 @@ test(Pred pred)
     Sequence<T2> expected(max_len);
     Sequence<T2> actual(max_len);
 
-    Sequence<T2> data(max_len, [&value](::std::size_t i) {
```

Comment:
Here we made the capturing by reference (unlike other places where you changed it to `[=]`) so in that line on code I would prefer `[&]`.